components/docs-chef-io/content/automate/backup.md

```diff
@@ -27,7 +31,7 @@ This amount of space needed for a backup varies depending on your Chef Automate
 * Elasticsearch snapshots of your Chef Automate configuration and data, such as converge, scan, and report data. You will need enough disk space for the each Elasticsearch snapshot and the delta--or the list of changes--for each successive snapshot
 * Chef Habitat Builder artifacts
 
-## Backup to a Filesystem
+## Set Up Backups to a Filesystem
```

Comment:
The content in the H2 sections about the various kinds of backups is more about how to set up backups to filesystem/aws/gcs than actually creating a backup.